### PR TITLE
Update bert.py

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Test

**Comment by wanda-carlson** [comment]
Test

File: src/bert.py [hunk 1]
```diff
@@ -9,5 +9,6 @@
 print("Hello Anna!")
 print("Hello Zak!")
 print("Hello Iffat")
+print("Hello Iffat")
```
Comment (bert-griffin) [inline src/bert.py:12]:
```suggestion
print("Hello Hey Hi")
```

**Review by bert-griffin — CHANGES_REQUESTED** [review]
Can you fix this!

File: src/bert.py [hunk 2]
```diff
@@ -9,5 +9,6 @@
 print("Hello Anna!")
 print("Hello Zak!")
 print("Hello Iffat")
+print("Hello Hey Hi")
```
Comment (bert-griffin) [inline src/bert.py:12]:
```suggestion
print("Hello Hey Hi")
print("But no Why! ")
```

**Review by bert-griffin — APPROVED** [review]

**Pull request merged**